modules/core/include/opencv2/core/optim.hpp

```diff
@@ -291,9 +292,13 @@ in the latter case it is understood to correspond to \f$c^T\f$.
 and the remaining to \f$A\f$. It should contain 32- or 64-bit floating point numbers.
 @param z The solution will be returned here as a column-vector - it corresponds to \f$c\f$ in the
 formulation above. It will contain 64-bit floating point numbers.
+@param constr_eps allowed numeric disparity for constraints
 @return One of cv::SolveLPResult
  */
-CV_EXPORTS_W int solveLP(InputArray Func, InputArray Constr, OutputArray z);
+CV_EXPORTS_W int solveLP(InputArray Func, InputArray Constr, OutputArray z, double constr_eps);
+
+/** @overload */
+CV_EXPORTS int solveLP(InputArray Func, InputArray Constr, OutputArray z);
```

Comment:
Do we need the overload for 4.x or default value for eps is enough?

Reply:
If we want to preserve binary compatibility with 4.7 then we need the overload. I vote for "yes" :slightly_smiling_face: